src/darker/black_diff.py

```diff
@@ -174,6 +178,11 @@ def run_black(src_contents: TextDocument, black_config: BlackConfig) -> TextDocu
     mode = BlackModeAttributes()
     if "line_length" in black_config:
         mode["line_length"] = black_config["line_length"]
+    if "target_version" in black_config:
+        ver_str = black_config["target_version"]
+        ver = next((v for v in TargetVersion if v.name.lower() == ver_str), None)
+        if ver:
+            mode["target_versions"] = {ver}
```

Comment:
This only adds support for `target-version =` in the configuration file, but doesn't add a corresponding `--target-version=` command line argument. This is inconsistent with other supported Black configuration options. See `make_argument_parser()` in `darker.command_line`.